tutorials/dataframe/df020_helpers.C

```diff
@@ -22,7 +22,7 @@ void df020_helpers()
    auto isOne = [] (int a) { return a == 1; };
    auto isNotOne = ROOT::RDF::Not(isOne);
 
-   // Both `isOne` and `isNotOne` are callables that we can use in `Filters`:
+   // Both `isOne` and `isNotOne` are callable that we can use in `Filters`:
```

Comment:
this is wrong?

Reply:
it is an adjective... adjectives to do have plural in english .. asaik....

Reply:
Or may be they are nouns in that case...

Reply:
let me fix

Reply:
changed back.